src/rmc-compiler/build.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+
+macro_rules! path_str {
+    ($input:expr) => {
+        &$input.iter().collect::<PathBuf>().to_string_lossy().into_owned()
+    };
+}
+
+/// Build the target library, and setup cargo to rerun them if the source has changed.
+fn setup_lib(out_dir: &str, lib_out: &str, lib: &str) {
+    let rmc_lib = vec!["..", "..", "library", lib];
```

Comment:
why `..` twice? Should one be `.`

Reply:
Because this is inside src/rmc-compiler/ so we need to walk back two directories.

Reply:
Ahh, I thought this was a search path, not a set of folders